src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala

```diff
@@ -181,8 +214,18 @@ class ClusteredBeam[EventType: Timestamper, InnerBeamType <: Beam[EventType]](
       tuning.segmentBucket(now - tuning.windowPeriod).start,
       tuning.segmentBucket(now + tuning.windowPeriod).end
     )
+    log.debug("Looking for beam having start time [%s]", timestamp)
     val futureBeamOption = beams.get(timestamp.millis) match {
       case _ if !open => Future.value(None)
+      // If the granularity has changed since tranquility restart and we found some beam that can handle this event return that beam
```

Comment:
It would be nice if you could truncate these lines down to make it more likely be able to read without scrolling.  Generally speaking, ~120 characters is a good cutoff point.  You should be able to configure your IntelliJ to draw a line at the 120 character mark so that you know where that is.